app/models/tag.rb

```diff
@@ -368,6 +368,9 @@ def async(method, *args)
 
   # Class methods
 
+  def self.tag_feeds_key(id)
+    "/v1/rss_data_tags/#{id}"
```

Comment:
are these paths going to be user-visible or are they intended to only ever be seen by readers? why rss_data_tags -- any reason not to just have rss_tags or tags_rss? Why is the v1 in the url, would we ever have multiple versions active? (Not that we shouldn't, just want to know the reasoning.)